lua/telescope/builtin/internal.lua

```diff
@@ -826,7 +834,6 @@ internal.keymaps = function(opts)
     end,
   }):find()
 end
-
```

Comment:
we are missing a newline :sob:

Reply:
What have I done? :scream: and why does neither luacheck or stylua check complain (and ofc my stylua not take care of this)